repos/core-setup.proj

```diff
@@ -4,8 +4,10 @@
   <PropertyGroup>
     <BuildArguments>-ConfigurationGroup=$(Configuration) -PortableBuild=$(PortableBuild) -SkipTests=true </BuildArguments>
     <BuildArguments Condition="$(Platform.Contains('arm'))">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
+    <BuildArguments Condition="'$(OS)' != 'Windows_NT' AND '$(Platform)' == 'x86'">$(BuildArguments) -TargetArchitecture=$(Platform) -DisableCrossgen=true -CrossBuild=true</BuildArguments>
     <BuildCommand>$(ProjectDirectory)/build$(ShellExtension) $(BuildArguments) -- /p:BuildDebPackage=false /p:BuildAllPackages=true /p:PreReleaseLabel="preview"</BuildCommand>
     <BuildCommand Condition="$(Platform.Contains('arm'))">$(ArmEnvironmentVariables) $(BuildCommand)</BuildCommand>
+    <BuildCommand Condition="'$(OS)' != 'Windows_NT' AND '$(Platform)' == 'x86'">$(CrossRootFsEnvironmentVariables) $(BuildCommand)</BuildCommand>
```

Comment:
I imagine these two lines could be made like this with the `RootFsEnvironmentVariables` rename...

```xml
<BuildCommand Condition="'$(RootFsEnvironmentVariables)' != ''">$(RootFsEnvironmentVariables) $(BuildCommand)</BuildCommand>
```

I'm not familiar with this scenario, so maybe there's some reason that doesn't work out I'm not aware of. Using the existence of `RootFsEnvironmentVariables` rather than checking conditions again seems nice to me though.

Reply:
Yes, this seems a better way